src/transformers/models/cohere/modeling_cohere.py

```diff
@@ -89,8 +88,6 @@ def forward(self, hidden_states):
         variance = (hidden_states - mean).pow(2).mean(-1, keepdim=True)
         hidden_states = (hidden_states - mean) * torch.rsqrt(variance + self.variance_epsilon)
         hidden_states = self.weight.to(torch.float32) * hidden_states
-        if self.bias is not None:
```

Comment:
I am guessing this was never used?

Reply:
This layer norm falls back to being `# Copied from Llama`

Reply:
Also missing on the `def rotate_half(x):` function

Reply:
This is different from Llama's Norm with is RMS and at different precision.

Reply:
Yep sorry